src/main/docker/artemis/artemis.yml

```diff
@@ -0,0 +1,42 @@
+# ----------------------------------------------------------------------------------------------------------------------
+# Artemis base service
+# ----------------------------------------------------------------------------------------------------------------------
+
+services:
+    artemis-app:
+        container_name: artemis-app
+        extra_hosts:
+            - "host.docker.internal:host-gateway"
```

Comment:
Why this extra host?

Reply:
Afaik this is still[ the workaround to make `host.docker.internal` working on Linux](https://github.com/docker/for-linux/issues/264#issuecomment-784985736). This is for example necessary if you have some external services hosted on your docker host and want to reach them from within the container.

Reply:
But I don't get why we need it here. You can always use http://artemis-app:8080 or similar endpoints :) At least that's what we are doing in our production systems

Reply:
If everything runs inside the docker network I use `artemis-app`, but if I want to connect to an external system like Jira running in a VM on the host I will use `host.docker.internal`. I expect this to be mainly used by developers and not necessarily in production systems.

Reply:
Ah .. now i get it. In this case `localhost` will not work, right ?

In this case I would suggest to move the extra_hosts to the development part :)

Reply:
Yes because `localhost` will point to the docker container and not the docker host which is possible with `host.docker.internal`. Just moved it to the development version of the docker compose file.